Makefile

```diff
@@ -1,12 +1,15 @@
 .PHONY: default
 default: help
 
+include concourse/content-event-resource/Makefile
+
 .PHONY: help
 help:
 	@echo "make help              Show this help message"
 	@echo "make services          Run the services that Concourse and the pipeline requires"
 	@echo "make initdb            Restore DB dump on first run, can be also used to restore DB"
 	@echo "make sql               Connect to the cnx-db database with a psql shell"
+	@$(MAKE) help_submake --no-print-directory
```

Comment:
Does this work if there is more than one submake?

Reply:
if more are added you need to make them unique... for example if there is another Makefile you'd have to change this to `help_event_resource_submake`.

Reply:
`make build-image` doesn't work in the root directory:

```
karen@yachiyo:~/src/rap-spike-concourse$ make build-image
python setup.py sdist
python: can't open file 'setup.py': [Errno 2] No such file or directory
concourse/content-event-resource/Makefile:20: recipe for target 'build-sdist' failed
make: *** [build-sdist] Error 2
```

May be we shouldn't just include the commands.